helper/configure-image.sh

```diff
@@ -1,20 +1,24 @@
 #!/usr/bin/env sh
-# Do not us bash. Not all containers have bash installed
-set -ex
+# Use a generic shell instead of bash for compatibility across different containers
 
-# Check configuration
-[ -z "$DDE_UID" ] && echo "DDE_UID is not set" && exit 1;
-[ -z "$DDE_GID" ] && echo "DDE_GID is not set" && exit 1;
+set -ex
+# -e: Exit immediately if a command exits with a non-zero status
+# -x: Print commands and their arguments as they are executed
 
-###############################################################################
+# Check if necessary environment variables are set
+[ -z "$DDE_UID" ] && echo "DDE_UID is not set" && exit 1
+[ -z "$DDE_GID" ] && echo "DDE_GID is not set" && exit 1
+[ -z "$DDE_CONTAINER_SHELL" ] && echo "DDE_CONTAINER_SHELL is not set" && exit 1
```

Comment:
@sbaerlocher why exit here if you define another default a couple lines down? Also we got multiple default values in exec.sh:30+31, dde.sh:17

Reply:
The line is removed, and within the script itself, `sh` is set if it is not provided through Docker Compose.